source/scss/stinson.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ * Stinson
+ *
+ */
+
+@import 'shared';
+
+// mixin to extend stinson filter
+// @mixin stinson
+// @param $filters... {filter} - Zero to many css filters to be added
+// @example
+//   img {
+//     @include stinson;
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px));
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px)) {
+//       /*...*/
+//     };
+//   }
+@mixin stinson($filters...) {
+  @include filter-base;
+  filter: contrast(0.75) saturate(0.85) $filters;
+
+  @content;
```

Comment:
I would also add a pseudo element with a light orange light to bring up the warmth in the shadows. This is what I tried and it looked pretty close:

```
&::before {
  background: rgba(240, 149, 128, .2);
  mix-blend-mode: soft-light;
}
```

Before changes:

![screen shot 2016-11-02 at 10 13 59 am](https://cloud.githubusercontent.com/assets/1693164/19932101/29fe7ecc-a0e5-11e6-835a-6b718af71343.jpg)


After:

![screen shot 2016-11-02 at 10 13 49 am](https://cloud.githubusercontent.com/assets/1693164/19932095/24c5b128-a0e5-11e6-9593-3b98f4b196d6.jpg)